Chore/AssignedChore.js

```diff
@@ -0,0 +1,451 @@
+ //@CheckFlag(AssignedChoreFirstTime) @SetFlag(AssignedChoreFirstTime)
+if(!isVar("FirstassignedChoreRun")){

```

Comment:
What is this file? I already have all of that inside the Chore file as far as I can tell.
Point out what is new and I can try to merge this stuff into my Chore.js

Reply:
its from the original working chore routine... it supports assigned chore flow.. (I think you clobbered this code when you restarted chores, but didn't implement it).. I merged and rehooked chore menu to support it...  if you want to keep refactoring stuff you should test before pushing up..